config/config.go

```diff
@@ -23,5 +23,7 @@ func init() {
 
 // Prefixes used in Mist stream names to let us determine whether a given "stream" in Mist is being used
 // for the segmenting or transcoding phase
+const SEGMENTING_PREFIX = "catalyst_vod_"
 const SOURCE_PREFIX = "tr_src_"
 const RENDITION_PREFIX = "tr_rend_+"
+const RECORDING_PREFIX = "video"
```

Comment:
Should we call this "live"? Would it be clearer to indicate it's a live stream?

Reply:
Yes, makes sense. We need to change this in live workflow and then update in catalyst API.